.mergify.yml

```diff
@@ -30,10 +30,12 @@ pull_request_rules:
           - and:
             - author=mozilla-l10n-automation-bot
             - base=main
-            - status-success=pr-complete
+            - head=import-l10n
```

Comment:
⚠️ I'm still unsure about this line. I don't know if the author need to be included in the branch name. I haven't set up the l10n bot on my fork yet.

Reply:
We may be able to figure this out by looking at the tasks created by https://firefox-ci-tc.services.mozilla.com/hooks/project-releng/cron-task-mozilla-l10n-android-l10n-tooling%2Fupdate-l10n ?

Reply:
Tested at https://github.com/mozilla-releng/staging-fenix/pull/10. `import-l10n` is the right value.

For the record, I used this hook[1] instead. `update-l10n` updates the `android-l10n` repo instead of the product ones. I don't know the process but it seems there's some back-and-forth between `android-l10n` and the product repos and one hook goes one way, the other one goes the other way. I any case, I manually crafted this TC task[3] so that it targets our staging fork instead of the main repo. Since there's not level 1 secrets here, I used the level 3 ones. That's the only level-3 thing I used.

[1] https://firefox-ci-tc.services.mozilla.com/hooks/project-releng/cron-task-mozilla-l10n-android-l10n-tooling%2Fupdate-projects
[2] e.g.: https://github.com/mozilla-l10n/android-l10n/pull/473
[3] https://firefox-ci-tc.services.mozilla.com/tasks/ENulF1PySFGvR_p91ohwJQ